update for range annotation (#2734)
influxdata/docs-v2: content/influxdb/cloud/visualize-data/annotations.md

Annotations may be useful to highlight operations or anomalies for your team to reference.
{{% /note %}}

<!-- {{< youtube 5NEplCesNAc >}} --->

## Create an annotation

1. In the navigation menu on the left, select **Boards** (**Dashboards**).

    {{< nav-icon "dashboards" >}}

> 2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/). The **Annotations** option is selected by default.

Reviewer: @daviddkkim didn't this recently change with the annotations onboarding changes you made?

Reviewer 2: yes, annotations are defaulted off ! Sorry for the late reply

Author: ```suggestion
2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/), and then click the **Annotations** button.
```

> 3. In a dashboard cell, doing one of the following:

Reviewer: Is `doing` the right form here? Should it be `do`? None of the other verbs are in present participle

Author: ```suggestion
3. In a dashboard cell, do one of the following:
```

>    - To add an annotation to a single point in time, press Shift and click the time.

Reviewer: should this be something more like "click the graph at the time you want to annotate" ?

Author: ```suggestion
   - Press Shift and click the graph at the time you want to annotate.
```
